homeassistant/components/slack/switch.py

```diff
@@ -0,0 +1,48 @@
+"""Slack platform for switch component."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import SlackEntity
+from .const import DATA_CLIENT, DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Slack switch."""
+    async_add_entities(
+        [
+            SlackSwitchEntity(
+                hass.data[DOMAIN][entry.entry_id][DATA_CLIENT],
+                SwitchEntityDescription(
+                    key="status",
+                    name="Status",
```

Comment:
Should we call it `Status` if it's really an away switch ?

Reply:
Or should this better be a `select` so it's clear what options exist?

Reply:
`Active` sounds better. There is only 2 options so a switch should be appropriate?

Reply:
The switch is not clear to what each option means, since it switches between "Away" and "Auto".